kolomiets/nicofog/src/main/java/service/UserService.java

```diff
@@ -0,0 +1,58 @@
+package service;
+
+import dao.mysql.UserRepository;
+import entity.User;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+/**
+ * Created by mihail on 11/10/18.
+ */
+public class UserService {
+
+    private UserRepository userRepository;
+    private String resultMessage;
+
+    public void addUser(User user) {
+        userRepository = new UserRepository();
+
+        user.setRole("user");
+        user.setDateRegistration(LocalDateTime.now().toString());
+        user = userRepository.add(user);
+        
+        if (user == null) {
+            resultMessage = "Can't add user";
+        } else {
+            resultMessage = user.getName() + " added success";
+        }
+    }
+
+    public List<User> getAll() {
+        userRepository = new UserRepository();
```

Comment:
That's what I'm talking about. You create instance of service for every operation.